File: exemplos/operacoes_es/Soma.java
// Programa Soma gerado automaticamente por PIFC

import java.util.Scanner;

public class Soma {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		double a, b, soma;
		System.out.println("Digite dois numeros:");
		a = scanner.nextDouble();
		b = scanner.nextDouble();
		soma = a + b;
		System.out.println("A soma deles eh : " + Double.toString(soma));
	}

}

File: exemplos/operacoes_es/Soma.c
// Programa Soma gerado automaticamente por PIFC

#include <stdio.h>
#include <string.h>
#include <stdlib.h>

typedef unsigned char string[1024];
typedef int bool;
enum {false, true};

int main(int argc, char *argv[]){
	double a, b, soma;
	printf("%s\n", "Digite dois numeros:");
	scanf("%lf %lf", &a, &b);
	soma = a + b;
	printf("%s%lf\n", "A soma deles eh : ", soma);
	return EXIT_SUCCESS;
}
